themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2529,6 +2529,610 @@ Status: 200 OK
+| `stack`             | string | path  | stack name                                                                                                   |
+| `timestamp`         | string | body  | ISO timestamp of when to destroy the stack                                                                   |
+| `deleteAfterDestroy`| bool   | body  | whether to delete the stack after resources are destroyed                                                    |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  --request POST \
+  --data '{"timestamp":"{timestampValue}","deleteAfterDestroy":"{deleteAfterDestroyValue}"}' \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/deployments/ttl/schedules
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "id": "12345678-8102-447f-b246-e9ec85786e23",
+  "orgID": "87654321-8b3b-418a-8c01-5ddd0e00bace",
+  "scheduleCron": "0 0 * * *",
+  "nextExecution": "2024-04-20 00:00:00.000",
+  "paused": false,
+  "kind": "deployment",
+  "definition": {
+    "programID": "12345678-9f94-4a92-8bfb-c2da724fa65b",
+    "request": {
+        "inheritSettings": true,
+        "operation": "destroy",
+        "operationContext": {
+            "options": {
+                "deleteAfterDestroy": true
+            }
+        }
+    }
+  },
+  "created": "2024-04-19 01:00:00.000",
+  "modified": "2024-04-19 01:00:00.000",
+  "lastExecuted": null
+}
+```
+
+### Create Raw Operation Schedule
+
+```
+POST /api/stacks/{organization}/{project}/{stack}/deployments/schedules
+```
+
+#### Parameters
+
+| Parameter           | Type   | In    | Description                                                                                                  |
+|---------------------|--------|-------|--------------------------------------------------------------------------------------------------------------|
+| `organization`      | string | path  | organization name                                                                                            |
+| `project`           | string | path  | project name                                                                                                 |
+| `stack`             | string | path  | stack name                                                                                                   |
+| `scheduleCron`      | string | body  | cron expression for when to run the pulumi operation                                                         |
+| `scheduleOnce`      | string | body  | ISO timestamp of when to destroy the stack                                                                   |
+| `operation`         | string | body  | pulumi operation to perform. Can be "update", "destroy", "refresh" or "preview"                              |
```

Comment:
This should be `request` instead of `operation` I think?
```suggestion
| `request`         | [CreateDeploymentRequest](/docs/pulumi-cloud/deployments/api/#create-deployment) | body  | The create deployment request object that will be executed on every invocation |
```

Reply:
I was aiming to make it simpler for users, and to match UI, but yeah, ideally we should point them to full range of things for the raw one, updating to this.